OloEngine/tests/Rendering/WaterRenderingTest.cpp

```
@@ -144,6 +197,50 @@ TEST(WaterRendering, WaterComponentDefaults)
+    EXPECT_FLOAT_EQ(wc.m_RefractionDistortion, 0.05f);
+    EXPECT_FLOAT_EQ(wc.m_RefractionHeightFactor, 0.5f);
+    EXPECT_FLOAT_EQ(wc.m_RefractionColor.r, 0.0f);
+    EXPECT_FLOAT_EQ(wc.m_RefractionColor.g, 0.05f);
+    EXPECT_FLOAT_EQ(wc.m_RefractionColor.b, 0.1f);
+
+    // Foam defaults
+    EXPECT_EQ(wc.m_FoamTexture, 0u);
+    EXPECT_FLOAT_EQ(wc.m_FoamHeightStart, 0.3f);
+    EXPECT_FLOAT_EQ(wc.m_FoamFadeDistance, 0.5f);
+    EXPECT_FLOAT_EQ(wc.m_FoamTiling, 2.0f);
+    EXPECT_FLOAT_EQ(wc.m_FoamBrightness, 1.5f);
```

Cppcheck (2.20.0)

[style] The function 'MakeTestRNG' is never used.

(unusedFunction)